src/utils/importClassesFromDirectories.ts

```diff
@@ -0,0 +1,16 @@
+import * as path from 'path';
+
+/**
+ * Loads all exported classes from the given directory.
+ */
+export function importClassesFromDirectories(directories: string[], formats = ['.ts']): string[] {
+  const allFiles = directories.reduce((allDirs, dir) => {
+    return allDirs.concat(require('glob').sync(path.normalize(dir)));
+  }, [] as string[]);
+
+  return allFiles
```

Comment:
I don’t see anything in here that checks if zero files were found. I think it’s important that we throw an error when the glob doesn’t find any files because that will inform the user that the glob is probably incorrect. Otherwise, the user will wonder why their routes.ts was generated without any routes.

I’d also like to see an automated unit test or an integration test that confirms this.